### PR TITLE
Adds Monitoring with Telegraf guide

### PR DESCRIPTION
**Pull request by petems**
* Installing Telegraf
* Configuring Telegraf
* Configuring Consul to send metrics to Telegraf
* Important metrics and aggregates from Telegraf plugin

### CODE REVIEW

File: website/source/docs/guides/monitoring-telegraf.html.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        consul.server2.raft.apply
+        consul.server3.raft.apply
+
+If you are using a different agent (e.g. Circonus, Statsite, or plain statsd), you can find the configuration reference [here][consul-telemetry-config].
+
+## Visualising Telegraf Consul Metrics
+
+There a number of ways of consuming the information from Telegraf.
+
+In the [consul-guides repository][consul-guides repository], we have an example of a 3 Node Consul Cluster with a Vault instance, sending metrics to InfludDB via Telegraf, visualized using Grafana.
```
Comment (petems) [inline website/source/docs/guides/monitoring-telegraf.html.md]: Can we directly link to the consul-guides repo? If so, should we wait until my PR over there is merged?

**Comment by petems** [comment]
Vagrant guide: https://github.com/hashicorp/consul/pull/4227

File: website/source/docs/guides/monitoring-telegraf.html.md (hunk 1 above)
Comment (pearkes) [reply website/source/docs/guides/monitoring-telegraf.html.md]: I actually think we could avoid this, as we're not explicitly tasked with maintaining it in concert with the documentation. I think it's best to keep all the content inline as it is below and not link externally. It makes it easier for the community to contribute to it all in one place, as well.

Comment (petems) [reply website/source/docs/guides/monitoring-telegraf.html.md]: So, should I put the Vagrant configuration in the Consul repo here and remove it from the guides, or just remove the reference completely?

Comment (petems) [reply website/source/docs/guides/monitoring-telegraf.html.md]: As discussed later, we removed the direct reference, we can follow up when the change is merged into https://github.com/hashicorp/consul-guides/pull/17

**Pull request merged**